lib/screens/home/home_screen.dart

```diff
@@ -1,18 +1,185 @@
 import 'package:flutter/material.dart';
+import 'package:page_view_dot_indicator/page_view_dot_indicator.dart';
+import 'package:survey_flutter/gen/assets.gen.dart';
 
 import 'home_shimmer_loading.dart';
 
 const routePathHomeScreen = '/home';
 
-class HomeScreen extends StatelessWidget {
+class HomeScreen extends StatefulWidget {
   const HomeScreen({super.key});
 
+  @override
+  State<StatefulWidget> createState() => HomeScreenState();
+}
+
+class HomeScreenState extends State<HomeScreen> {
+  TextTheme get _textTheme => Theme.of(context).textTheme;
+
+  final _pageController = PageController();
+  final int _pageCount = 2;
+
+  bool _isLoading = true;
+  int _selectedPage = 0;
+
+  @override
+  void initState() {
+    super.initState();
+    // TODO remove mock duration in integration ticket
+    Future.delayed(const Duration(seconds: 3), () {
+      setState(() {
+        _isLoading = false;
+      });
+    });
+  }
+
   @override
   Widget build(BuildContext context) {
-    // TODO Add home screen UI in part 2
-    return const Scaffold(
-      backgroundColor: Colors.black,
-      body: HomeShimmerLoading(),
+    // TODO modify the loading in integration ticket
+    if (_isLoading) {
+      return const HomeShimmerLoading();
+    } else {
+      return _buildHomeScreenContent();
+    }
+  }
+
+  Widget _buildHomeScreenContent() {
+    // TODO bind with real data in integrate ticket
+    return Scaffold(
+      body: Stack(
+        alignment: Alignment.center,
+        fit: StackFit.expand,
+        children: [
+          PageView(
+            controller: _pageController,
+            onPageChanged: (page) {
+              setState(() {
+                _selectedPage = page;
+              });
+            },
+            children: [_buildBackgroundImage(), _buildBackgroundImage()],
+          ),
+          Align(
+            alignment: Alignment.topLeft,
+            child: Padding(
+              padding: const EdgeInsets.fromLTRB(20, 60, 0, 0),
```

Comment:
```suggestion
              padding: const EdgeInsets.only(left: 20, top: 60),
```
And we should extract the dimen to a variable instead of hardcoding here